apis/generate.go

```diff
@@ -22,8 +22,14 @@ limitations under the License.
 // Remove existing CRDs
 //go:generate rm -rf ../config/crd
 
+// TODO(hasheddan): update following enabling version.
```

Comment:
Note: generation is skipping over `v1beta1` for the time being

Reply:
I'd suggest adding some more context to this TODO if we expect it to get merged - it's not quite clear what it means.